tests/robot-tests/tests/libs/common.robot

```diff
@@ -555,3 +556,20 @@ user checks page contains other release
 user checks page does not contain other release
     [Arguments]   ${other_release_title}
     user checks page does not contain element   xpath://li[@data-testid="other-release-item"]/a[text()="${other_release_title}"]
+
+user navigates to find statistics page 
+    environment variable should be set   PUBLIC_URL
+    user goes to url   %{PUBLIC_URL}/find-statistics
+    user waits until h1 is visible  Find statistics and data
+    user waits for page to finish loading
+
+
+user navigates to publish release on public frontend 
```

Comment:
I'd remove this keyword, as it's only two lines and I think you're only using it once.